docs/docs/cmd/spo/list/list-roleinheritance-reset.md

```diff
@@ -0,0 +1,26 @@
+# spo list roleinheritance reset
+
+Restore role inheritance on list or library.
+
+## Usage
+
+```sh
+m365 spo list roleinheritance reset [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the list is located
+
+`-i, --listId [listId]`
+: ID of the list. Specify either id or title but not both
+
+`-t, --listTitle [listTitle]`
+: Title of the list. Specify either id or title but not both
+
+
```

Comment:
Let's remove this unnecessary empty line

Reply:
done